### PR TITLE
Docs/add zapier

### PR DESCRIPTION
**Pull request by jobenjada**
## What does this PR do?

- adds zapier tutorial to docs incl new category
- opens all links within a docs page in a new tab

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [X] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

Read and test a few links.

## Checklist

Yes

### CODE REVIEW

File: apps/formbricks-com/pages/docs/integrations/zapier/index.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+<Image src={ChooseEvent} alt="Choose Event" quality="100" className="rounded-lg" />
+
+## Step 5: Connect Formbricks with Zapier
+
+Now, you have to connect Zapier with Formbricks via an API Key:
+
+<Image src={ConnectWithFB1} alt="Connect with Formbricks - 1" quality="100" className="rounded-lg" />
+<Image src={ConnectWithFB2} alt="Connect with Formbricks - 2" quality="100" className="rounded-lg" />
+
+Go back to your Formbricks → Settings → API Keys and create an API key for the environment you’re working with: When you **don’t** see an orange bar at the top of your app, you are in Production Mode hence you need a Production Key:
```
Comment (mattinannt) [inline apps/formbricks-com/pages/docs/integrations/zapier/index.mdx:71]: I would move this to the `API Key Setup page` and reference/link it on the zapier docs page.

File: apps/formbricks-com/pages/docs/integrations/zapier/index.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+<Callout title="Already published? Duplicate survey" type="note">
+  You can only update the questionId when the survey was not yet published. Already published it? Just
+  **duplicate it** to update the questionIds.
+  <Image src={DuplicateSurvey} alt="Duplicate Survey" quality="100" className="rounded-lg" />
+</Callout>
+
+## Step 3: Send a test response
+
+In order to set up Zapier you’ll need a test response. This allows you to select the individual values of each response in your Zap. If you want to set up an in-app survey, you can use our [Demo App](/docs/contributing/demo) to trigger a survey and submit a response that without much setting. It also works with Formbricks Cloud.
```
Comment (mattinannt) [inline apps/formbricks-com/pages/docs/integrations/zapier/index.mdx:50]: I wouldn't mention the Demo app here as it is no option for most of the users (who haven't setup the repository and/or have no knowledge how to do so).

File: apps/formbricks-com/pages/docs/integrations/zapier/index.mdx (hunk 1 above)
Comment (jobenjada) [reply apps/formbricks-com/pages/docs/integrations/zapier/index.mdx]: makes sense, updated

File: apps/formbricks-com/pages/docs/integrations/zapier/index.mdx (hunk 2 above)
Comment (jobenjada) [reply apps/formbricks-com/pages/docs/integrations/zapier/index.mdx]: not sure if it helps to not inlude any info about how to get a test submission for an in-app survey. I have reworded it a bit :)

**Pull request merged**